packages/mds-db/policies.ts

```diff
@@ -218,7 +218,9 @@ export async function publishPolicy(policy_id: UUID) {
       throw new NotFoundError('cannot publish nonexistent policy')
     }
 
-    const publish_date = now()
+    if (!policy.start_date || policy.start_date < publish_date) {
```

Comment:
A policy should always have a `start_date` -- it shouldn't even be able to get past the validation layer in the API without it.

Reply:
What if you haven't published the policy yet and you haven't decided what the `start_date` should be?

Reply:
Ok, after discussing with @avatarneil , I still think being able to save a half-edited Policy in the DB would be good for the user, but since we make other fields mandatory right now, I'll make start_date mandatory.